src/panels/lovelace/components/hui-entity-editor.ts

```diff
@@ -76,10 +76,11 @@ export class HuiEntityEditor extends LitElement {
   protected _optionChanged(ev: Event): void {
     const target = ev.target! as EditorTarget;
```

Comment:
Generally it's better to use `ev.currentTarget`, as that is the element you attached your handler to. `ev.target` could be a smaller element living inside the element you had your event listener attached to.